src/Stash.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Transposit Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Transposit } from ".";
+
+export class Stash {
+  private transposit: Transposit;
+  constructor(transposit: Transposit) {
+    this.transposit = transposit;
+  }
+
+  async listKeys(): Promise<string[]> {
+    return this.transposit.makeCallJson<string[]>("GET", "/api/v1/stash/keys");
+  }
+
+  async get(key: string): Promise<any> {
+    const queryParams = { keyName: key };
+    return await this.transposit.makeCallJson<any>(
+      "GET",
+      "/api/v1/stash/value",
+      { queryParams },
+    );
+  }
+
+  async put(key: string, value: any): Promise<void> {
+    const queryParams = { keyName: key };
+    await this.transposit.makeCall("POST", "/api/v1/stash/value", {
+      queryParams,
+      body: value,
+    });
+    return;
+  }
+
+  async remove(key: string): Promise<void> {
```

Comment:
as mentioned here https://github.com/transposit/transposit-js-sdk/pull/13#discussion_r329795833, I don't think we should have an explicit `remove()` method. I think we should try to be consistent with the javascript API in the backend.